src/cljs/reagent_toolbox_docs/components/drawer.cljs

```diff
@@ -0,0 +1,102 @@
+;;;; Copyright © 2017 Flexpoint Tech Ltd
+
+(ns reagent-toolbox-docs.components.drawer
+  (:require [reagent-toolbox-docs.layout :as layout]
+            [reagent-toolbox-docs.ui :as ui]))
+
+(defn view
+  []
+  (fn []
+    [:article
+     [:h1 "Drawer"]
+     [:p "The "
```

Comment:
Not sure what you're on about here. I generally limit line length to 79 characters so I split stuff over. I can break that habit, surely, but I don't see a problem here with dividing `[:p]` lines to fit into 79 column chunks.